tests/test_fields.py

```diff
@@ -378,7 +378,7 @@ def test_empty_fields_serialized_with_defaults(self):
         second_schema = self.set_up_schema(None)
         second_source = second_schema()
 
-        self.assertEqual({}, second_source.serialize())
+        self.assertEqual({}, second_source.serialize(implicit_nulls=True))
```

Comment:
this whole test has an outdated premise "empty fields serialized with defaults", can probably just remove it. or rewrite to communicate that the presence or absence of a default doesn't have any special behavior beyond the ordinary implicit_nulls behavior